### ISSUE TITLE
Module can't to delete\manage secondary_zone parameter.

### ISSUE THREAD

**Issue by ksemele** [terraform-google-modules/terraform-google-sql-db#353]
### TL;DR

After import PostgreSQL instance from google I can't manage one value:
```
resource "google_sql_database_instance" "default" {
      settings {
          location_preference {
              secondary_zone = "europe-west1-c"  # <-- that
            }
        }
    }
```
Actually, provider supports that as optional parameter:
https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance#secondary_zone
And that parameter setted up manually in google console

I try to backport them into my Terraform code, but can't find them as input parameter:
https://registry.terraform.io/modules/GoogleCloudPlatform/sql-db/google/latest/submodules/postgresql?tab=inputs

When I run `terraform apply` I see successful of process:
```
...
Releasing state lock. This may take a few moments...

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.

Outputs: ...
```

### Expected behavior

Terraform plan after apply will being show me 'no changes'

### Observed behavior

Terraform plan trying to delete `secondary_zone = "europe-west1-c" -> null` again...

### Terraform Configuration

```hcl
# https://registry.terraform.io/modules/GoogleCloudPlatform/sql-db/google/latest/submodules/postgresql
module "sql-db_postgresql" {
  source     = "GoogleCloudPlatform/sql-db/google//modules/postgresql"
  version    = "12.0.0"
  project_id = var.project

  database_version = "POSTGRES_13"
  name             = "my-name"
  region           = "europe-west1"
  zone             = "europe-west1-b"

  availability_type = "REGIONAL"


  backup_configuration = {
    enabled                        = true
    start_time                     = "04:00"
    location                       = "eu"
    point_in_time_recovery_enabled = false
    transaction_log_retention_days = null
    retained_backups               = 10
    retention_unit                 = null
  }

  tier          = "db-custom-8-16384"
  db_name       = "mybase"
  user_name     = var.psql_user
  user_password = var.psql_password

  database_flags = [
    # my flags
  ]

  additional_databases = [
    # my bases
  ]


  ip_configuration = {
    authorized_networks : [
      # my networks
    ]
    ipv4_enabled = true
    private_network : "projects/${var.host_project}/global/networks/${var.firewall_network}",
    require_ssl        = false
    allocated_ip_range = ""
  }
}

output "psql-ip" {
  value = module.sql-db_postgresql.instance_ip_address
}

```

### Terraform Version

```
1.3.1
```

### Additional information

Terraform plan output before and after Terraform apply
```
Terraform will perform the following actions:
  module.sql-db_postgresql.google_sql_database_instance.default will be updated in-place
  ~ resource "google_sql_database_instance" "default" {
        id                             = "my-id"
        name                           = "my-name"
        (14 unchanged attributes hidden)
      ~ settings {
            (10 unchanged attributes hidden)
          ~ location_preference {
              - secondary_zone = "europe-west1-c" -> null
                (1 unchanged attribute hidden)
            }
            (13 unchanged blocks hidden)
        }
        (1 unchanged block hidden)
    }
Plan: 0 to add, 1 to change, 0 to destroy.
```

**Comment by Malik-Muneeb** [comment]
I'm also facing the same issue.

**Comment by krupnikola** [comment]
me too 👍

**Comment by KelvinVenancio** [comment]
same here

**Comment by KelvinVenancio** [comment]
Also, the PR with the fix is already open: https://github.com/terraform-google-modules/terraform-google-sql-db/pull/339

**Comment by g-awmalik** [comment]
#339 has been merged.

**Comment by xueshanf** [comment]
Is it possible to add  secondary_zone to v10.1.0? We cannot migrate to v.11.x just yet because the incompatible changes (e.g. password). Thanks!
